src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -95,7 +95,15 @@ if [ "$(uname -s)" == "Darwin" ]; then
 elif [ "$(uname -s)" == "Linux" ]; then
   # On Linux, we'll enable core file generation unconditionally, and if a dump
   # is generated, we will print some useful information from it and delete the
-  # dump immediately. 
+  # dump immediately.
+
+  if [ -e /proc/self/coredump_filter ]; then
```

Comment:
> Also as far as I can tell we are only doing the dumps on unix right now are there plans to add similar dump capturing on windows?

Dumps on Windows have been enabled via a registry change in the Server 2012 CI image (@mmitche did this recently).